src/views/components/EnterClaimsEvaluationData/index.tsx

```diff
@@ -0,0 +1,158 @@
+  value: string;
+  text: string;
+}
+
+interface ClaimTypeOptions {
+  key: string;
+  value: string;
+  text: string;
+}
+
+export interface State {
+  claims: ClaimsEvaluation;
+}
+
+class EnterClaimsEvaluationData extends React.Component<Props, State> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      claims: {
+        claimType: "",
+        workflowNumber: null,
+        agent: "",
+        allParametersMet: false,
+        comments: "",
+        evaluator: "",
+        loading: this.props.loading
+      }
+    };
+  }
+
+  public submitForm = async (): Promise<void> => {
+    return this.props.onSubmit(this.state.claims);
+  };
+
+  public handleDropDownInput = (_event, { name, value }): void => {
+    const claims = { ...this.state.claims, [name]: value };
+    return this.setState({ claims });
+  };
+
+  public AgentOptions = (agents: string[]): AgentOptions[] => {
+    return agents.map(
+      (agent): AgentOptions => {
+        return {
+          key: agent,
+          value: agent.toLowerCase(),
+          text: agent
+        };
+      }
+    );
+  };
+
+  public ClaimTypeOptions = (claimTypes: string[]): ClaimTypeOptions[] => {
+    return claimTypes.map(
+      (claimType): ClaimTypeOptions => {
+        return {
+          key: claimType,
+          value: claimType.toLowerCase(),
+          text: claimType
+        };
+      }
+    );
+  };
+
+  public handleInput = (event): void => {
+    const value = event.target.value;
+    const name = event.target.name;
+    const changedClaim = { [value]: { value, name } };
+    const claims = { ...this.state.claims, ...changedClaim };
+    this.setState({ claims });
+  };
+
+  public render() {
+    return (
+      <Form loading={this.state.claims.loading} onSubmit={this.submitForm}>
+        <Form.Field>
+          <label>Agents</label>
+          <Dropdown
+            placeholder="Select Agent"
+            fluid={true}
+            search={true}
+            selection={true}
+            options={this.AgentOptions(this.props.agents)}
+            onChange={this.handleDropDownInput}
+            name="agents"
+          />
+        </Form.Field>
+        <Form.Field>
+          <label>Type of claim</label>
+          <Dropdown
+            placeholder="Select claim"
+            fluid={true}
+            search={true}
+            selection={true}
+            options={this.ClaimTypeOptions(this.props.claimTypes)}
+            onChange={this.handleDropDownInput}
+            required={true}
+            name="claim"
+          />
+        </Form.Field>
+        <Form.Field>
+          <label>All parameters met </label>
+          <Checkbox
+            label={this.state.claims.allParametersMet}
+            onChange={this.handleInput}
+          />
+        </Form.Field>
+        <Form.Field>
+          <label>work flow number</label>
+          <input
+            placeholder="Add workflow number"
+            value={this.state.claims.workflowNumber}
```

Comment:
you can by default set `this.state.claims.workflowNumber` to zero and then write something like.  which will make the input value to be empty as long as its zero
```suggestion
            value={this.state.claims.workflowNumber || ""}
```

Reply:
This works fine after having set workflowNumber to zeron in; 
```this.state = {
      claims: {
        claimType: "",
        workflowNumber: 0,
        agent: "",
        allParametersMet: false,
        comments: "",
        evaluator: "",
        loading: this.props.loading
      },
      dropdownError: null
    };```